diff --git a/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java b/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
index 6a3297f..ac2d1ea 100644
--- a/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
+++ b/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
@@ -19,6 +19,9 @@ import com.google.android.gms.auth.api.credentials.HintRequest;
 import com.google.android.gms.common.ConnectionResult;
 import com.google.android.gms.common.api.GoogleApiClient;
 
+import java.util.Random;
+import java.util.Set;
+
 final class PhoneNumberHelper {
 
     private static final int REQUEST_PHONE_NUMBER_REQUEST_CODE = 1;
@@ -102,14 +105,17 @@ final class PhoneNumberHelper {
 
     @NonNull
     private GoogleApiClient getGoogleApiClient(@NonNull final Context context, final Activity activity) {
-        if (mGoogleApiClient == null) {
+        Set<GoogleApiClient> allClients = GoogleApiClient.getAllClients();
+        if (!allClients.isEmpty()) {
+            mGoogleApiClient = allClients.iterator().next();
+        } else if (mGoogleApiClient == null) {
             GoogleApiClient.Builder builder = new GoogleApiClient.Builder(context);
             builder = builder.addConnectionCallbacks(mApiClientConnectionCallbacks);
             builder = builder.addApi(Auth.CREDENTIALS_API);
 
             if (activity instanceof FragmentActivity) {
                 final FragmentActivity fragmentActivity = (FragmentActivity) activity;
-                builder = builder.enableAutoManage(fragmentActivity, mApiClientOnConnectionFailedListener);
+                builder = builder.enableAutoManage(fragmentActivity, new Random().nextInt(100), mApiClientOnConnectionFailedListener);
             }
 
             mGoogleApiClient = builder.build();
