diff --git a/node_modules/react-native-color-matrix-image-filters/android/build.gradle b/node_modules/react-native-color-matrix-image-filters/android/build.gradle
index d0d36fb..782e405 100644
--- a/node_modules/react-native-color-matrix-image-filters/android/build.gradle
+++ b/node_modules/react-native-color-matrix-image-filters/android/build.gradle
@@ -24,9 +24,9 @@ android {
 }
 
 dependencies {
-    if (GradleVersion.current() >= GradleVersion.version('4.1')) {
+    //if (GradleVersion.current() >= GradleVersion.version('4.1')) {
         compileOnly "com.facebook.react:react-native:${_reactNativeVersion}"
-    } else {
-        compile "com.facebook.react:react-native:${_reactNativeVersion}"
-    }
+    //} else {
+    //    compile "com.facebook.react:react-native:${_reactNativeVersion}"
+    //}
 }
