diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/FileProvider.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/FileProvider.java
index 9c706e6..771c0a0 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/FileProvider.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/FileProvider.java
@@ -1,4 +1,4 @@
 package com.imagepicker;
 
-public class FileProvider extends android.support.v4.content.FileProvider {
+public class FileProvider extends androidx.core.content.FileProvider {
 }
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
index b4311eb..8d6f2d1 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
@@ -12,11 +12,11 @@ import android.net.Uri;
 import android.os.Build;
 import android.provider.MediaStore;
 import android.provider.Settings;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.annotation.StyleRes;
-import android.support.v4.app.ActivityCompat;
-import android.support.v7.app.AlertDialog;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.StyleRes;
+import androidx.core.app.ActivityCompat;
+import androidx.appcompat.app.AlertDialog;
 import android.text.TextUtils;
 import android.util.Base64;
 import android.util.Patterns;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerPackage.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerPackage.java
index 7114345..d0ec43e 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerPackage.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerPackage.java
@@ -1,6 +1,6 @@
 package com.imagepicker;
 
-import android.support.annotation.StyleRes;
+import androidx.annotation.StyleRes;
 
 import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.JavaScriptModule;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ResponseHelper.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ResponseHelper.java
index 4c7a974..042f08b 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ResponseHelper.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ResponseHelper.java
@@ -1,6 +1,6 @@
 package com.imagepicker;
 
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 
 import com.facebook.react.bridge.Arguments;
 import com.facebook.react.bridge.Callback;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/media/ImageConfig.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/media/ImageConfig.java
index 71258ff..7d12c48 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/media/ImageConfig.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/media/ImageConfig.java
@@ -1,7 +1,7 @@
 package com.imagepicker.media;
 
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.webkit.MimeTypeMap;
 
 import com.facebook.react.bridge.ReadableMap;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/OnImagePickerPermissionsCallback.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/OnImagePickerPermissionsCallback.java
index 4068288..12214bd 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/OnImagePickerPermissionsCallback.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/OnImagePickerPermissionsCallback.java
@@ -1,6 +1,6 @@
 package com.imagepicker.permissions;
 
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 import com.facebook.react.modules.core.PermissionListener;
 
 /**
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionUtils.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionUtils.java
index cb62cbf..b466985 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionUtils.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionUtils.java
@@ -2,9 +2,9 @@ package com.imagepicker.permissions;
 
 import android.app.Activity;
 import android.content.DialogInterface;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.v7.app.AlertDialog;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
 
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.ReadableNativeMap;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionsHelper.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionsHelper.java
index 2d89834..4615941 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionsHelper.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/permissions/PermissionsHelper.java
@@ -1,6 +1,6 @@
 package com.imagepicker.permissions;
 
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 
 /**
  * Created by rusfearuth on 03.03.17.
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ButtonsHelper.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ButtonsHelper.java
index d8ad306..d8596e0 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ButtonsHelper.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ButtonsHelper.java
@@ -1,7 +1,7 @@
 package com.imagepicker.utils;
 
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 
 import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.bridge.ReadableMap;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/MediaUtils.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/MediaUtils.java
index 2eedf7f..6dddd07 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/MediaUtils.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/MediaUtils.java
@@ -8,8 +8,8 @@ import android.media.ExifInterface;
 import android.media.MediaScannerConnection;
 import android.net.Uri;
 import android.os.Environment;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 import com.facebook.react.bridge.ReadableMap;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ReadableMapUtils.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ReadableMapUtils.java
index d53a641..24a195f 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ReadableMapUtils.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/ReadableMapUtils.java
@@ -1,6 +1,6 @@
 package com.imagepicker.utils;
 
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 import android.text.TextUtils;
 
 import com.facebook.react.bridge.ReadableMap;
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/RealPathUtil.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/RealPathUtil.java
index 45189ce..e8dca99 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/RealPathUtil.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/RealPathUtil.java
@@ -10,9 +10,9 @@ import android.provider.DocumentsContract;
 import android.provider.MediaStore;
 import android.content.ContentUris;
 import android.os.Environment;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.v4.content.FileProvider;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.content.FileProvider;
 
 import java.io.File;
 
diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/UI.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/UI.java
index da0d2df..8195d86 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/UI.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/utils/UI.java
@@ -3,9 +3,9 @@ package com.imagepicker.utils;
 import android.content.Context;
 import android.content.DialogInterface;
 import android.graphics.drawable.ColorDrawable;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.v7.app.AlertDialog;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
 import android.widget.ArrayAdapter;
 
 import com.facebook.react.bridge.ReadableMap;
diff --git a/node_modules/react-native-image-picker/ios/RNImagePicker.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist b/node_modules/react-native-image-picker/ios/RNImagePicker.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist
new file mode 100644
index 0000000..b7f1002
--- /dev/null
+++ b/node_modules/react-native-image-picker/ios/RNImagePicker.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>RNImagePicker.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>158</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
