diff --git a/node_modules/react-native-easy-toast/index.js b/node_modules/react-native-easy-toast/index.js
index 043b55f..fb67798 100755
--- a/node_modules/react-native-easy-toast/index.js
+++ b/node_modules/react-native-easy-toast/index.js
@@ -23,7 +23,12 @@ export const DURATION = {
     FOREVER: 0,
 };
 
-const {height, width} = Dimensions.get('window');
+let {height, width} = Dimensions.get('window');
+if (width > height) {
+    const temp = height;
+    height = width;
+    width = temp;
+}
 
 export default class Toast extends Component {
 
