diff --git a/node_modules/react-native-tus-client/android/build.gradle b/node_modules/react-native-tus-client/android/build.gradle
index 509d7c6..8f6be82 100644
--- a/node_modules/react-native-tus-client/android/build.gradle
+++ b/node_modules/react-native-tus-client/android/build.gradle
@@ -34,8 +34,8 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile group: 'org.jetbrains', name: 'annotations', version: '13.0'
+    implementation 'com.facebook.react:react-native:+'
+    implementation group: 'org.jetbrains', name: 'annotations', version: '13.0'
     implementation 'io.tus.java.client:tus-java-client:0.4.2'
     implementation 'io.tus.android.client:tus-android-client:0.1.9'
 }
