diff --git a/node_modules/react-native-iphone-x-helper/index.js b/node_modules/react-native-iphone-x-helper/index.js
index afd571f..daa2536 100644
--- a/node_modules/react-native-iphone-x-helper/index.js
+++ b/node_modules/react-native-iphone-x-helper/index.js
@@ -9,8 +9,10 @@ export function isIphoneX() {
         ((dimen.height === 780 || dimen.width === 780)
           || (dimen.height === 812 || dimen.width === 812)
           || (dimen.height === 844 || dimen.width === 844)
+          || (dimen.height === 852 || dimen.width === 852)
           || (dimen.height === 896 || dimen.width === 896)
-          || (dimen.height === 926 || dimen.width === 926))
+          || (dimen.height === 926 || dimen.width === 926)
+          || (dimen.height === 932 || dimen.width === 932))
     );
 }
 
