diff --git a/node_modules/react-native-step-indicator/StepIndicator.js b/node_modules/react-native-step-indicator/StepIndicator.js
index 4c52f02..2663cac 100644
--- a/node_modules/react-native-step-indicator/StepIndicator.js
+++ b/node_modules/react-native-step-indicator/StepIndicator.js
@@ -240,10 +240,11 @@ export default class StepIndicator extends Component {
   renderStepLabels = () => {
     const { labels, direction, currentPosition, renderLabel } = this.props
     var labelViews = labels.map((label, index) => {
-      const selectedStepLabelStyle =
-        index === currentPosition
+      let selectedStepLabelStyle =
+        index <= currentPosition 
           ? { color: this.state.customStyles.currentStepLabelColor }
           : { color: this.state.customStyles.labelColor }
+      
       return (
         <TouchableWithoutFeedback
           style={styles.stepLabelItem}
@@ -365,7 +366,7 @@ export default class StepIndicator extends Component {
             stepStatus: this.getStepStatus(position)
           })
         ) : (
-          <Text style={indicatorLabelStyle}>{`${position + 1}`}</Text>
+          <Text style={indicatorLabelStyle}>{``}</Text>
         )}
       </Animated.View>
     )
