diff --git a/node_modules/react-native-coverflow/lib/Item.js b/node_modules/react-native-coverflow/lib/Item.js
index 8a89556..b723b3b 100644
--- a/node_modules/react-native-coverflow/lib/Item.js
+++ b/node_modules/react-native-coverflow/lib/Item.js
@@ -1,125 +1,170 @@
-Object.defineProperty(exports,"__esModule",{value:true});var _jsxFileName='src/Item.js';var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _react=require('react');var _react2=_interopRequireDefault(_react);
-var _reactNative=require('react-native');
-var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}
-
-var styles=_reactNative.StyleSheet.create({
-container:{
-position:'absolute',
-left:0,
-top:0,
-right:0,
-bottom:0,
-justifyContent:'center',
-alignItems:'center'}});var
-
-
-
-Item=function(_Component){_inherits(Item,_Component);function Item(){_classCallCheck(this,Item);return _possibleConstructorReturn(this,(Item.__proto__||Object.getPrototypeOf(Item)).apply(this,arguments));}_createClass(Item,[{key:'getChildContext',value:function getChildContext()
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-{var _props=
-this.props,scroll=_props.scroll,position=_props.position;
-return{
-animatedPosition:scroll.interpolate({
-inputRange:[position-2,position-1,position,position+1,position+2],
-outputRange:[-1,-1,0,1,1]})};
-
-
-}},{key:'shouldComponentUpdate',value:function shouldComponentUpdate(
-
-nextProps){
-
-return nextProps.position!==this.props.position||
-nextProps.rotation!==this.props.rotation||
-nextProps.midRotation!==this.props.midRotation||
-nextProps.perspective!==this.props.perspective||
-nextProps.scaleDown!==this.props.scaleDown||
-nextProps.scaleFurther!==this.props.scaleFurther||
-nextProps.wingSpan!==this.props.wingSpan||
-nextProps.spacing!==this.props.spacing||
-nextProps.children!==this.props.children;
-}},{key:'render',value:function render()
-
-{var _props2=
-
-
-
-
-
-
-
-
-
-
-
-this.props,scroll=_props2.scroll,position=_props2.position,rotation=_props2.rotation,midRotation=_props2.midRotation,perspective=_props2.perspective,scaleDown=_props2.scaleDown,scaleFurther=_props2.scaleFurther,wingSpan=_props2.wingSpan,spacing=_props2.spacing,onSelect=_props2.onSelect;
-
-var style={
-transform:[
-{perspective:perspective},
-{
-translateX:scroll.interpolate({
-inputRange:[position-2,position-1,position,position+1,position+2],
-outputRange:[spacing+wingSpan,spacing,0,-spacing,-spacing-wingSpan]})},
-
-
-{
-scale:scroll.interpolate({
-inputRange:[position-2,position-1,position,position+1,position+2],
-outputRange:[scaleFurther,scaleDown,1,scaleDown,scaleFurther]})},
-
-
-{
-rotateY:scroll.interpolate({
-inputRange:[
-position-2,
-position-1,
-position-0.5,
-position,
-position+0.5,
-position+1,
-position+2],
-
-outputRange:['-'+
-rotation+'deg','-'+
-rotation+'deg','-'+
-midRotation+'deg',
-'0deg',
-midRotation+'deg',
-rotation+'deg',
-rotation+'deg']})}]};
-
-
-
-
-
-
-return(
-_react2.default.createElement(_reactNative.View,{pointerEvents:'box-none',style:styles.container,__source:{fileName:_jsxFileName,lineNumber:114}},
-_react2.default.createElement(_reactNative.TouchableWithoutFeedback,{onPress:function onPress(){return onSelect(position);},__source:{fileName:_jsxFileName,lineNumber:115}},
-_react2.default.createElement(_reactNative.Animated.View,{style:style,__source:{fileName:_jsxFileName,lineNumber:116}},
-this.props.children))));
-
-
-
-
-}}]);return Item;}(_react.Component);Item.propTypes={scroll:_propTypes2.default.instanceOf(_reactNative.Animated.Value).isRequired,position:_propTypes2.default.number.isRequired,children:_propTypes2.default.element.isRequired,wingSpan:_propTypes2.default.number.isRequired,spacing:_propTypes2.default.number.isRequired,rotation:_propTypes2.default.number.isRequired,midRotation:_propTypes2.default.number.isRequired,perspective:_propTypes2.default.number.isRequired,scaleDown:_propTypes2.default.number.isRequired,scaleFurther:_propTypes2.default.number.isRequired,onSelect:_propTypes2.default.func.isRequired};Item.childContextTypes={animatedPosition:_propTypes2.default.instanceOf(_reactNative.Animated.Interpolation)};exports.default=
-
-
-Item;
\ No newline at end of file
+import React, { Component } from 'react';
+import {
+  Animated,
+  TouchableWithoutFeedback,
+  View,
+  StyleSheet,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+  container: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    right: 0,
+    bottom: 0,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+class Item extends Component {
+  static propTypes = {
+    scroll: PropTypes.instanceOf(Animated.Value).isRequired,
+    position: PropTypes.number.isRequired,
+    children: PropTypes.element.isRequired,
+    wingSpan: PropTypes.number.isRequired,
+    spacing: PropTypes.number.isRequired,
+    rotation: PropTypes.number.isRequired,
+    midRotation: PropTypes.number.isRequired,
+    perspective: PropTypes.number.isRequired,
+    scaleDown: PropTypes.number.isRequired,
+    scaleFurther: PropTypes.number.isRequired,
+    onSelect: PropTypes.func.isRequired,
+  };
+
+  static childContextTypes = {
+    animatedPosition: PropTypes.instanceOf(Animated.Interpolation),
+  };
+
+  getChildContext() {
+    const { scroll, position } = this.props;
+    return {
+      animatedPosition: scroll.interpolate({
+        inputRange: [
+          position - 2,
+          position - 1,
+          position,
+          position + 1,
+          position + 2,
+        ],
+        outputRange: [-1, -1, 0, 1, 1],
+      }),
+    };
+  }
+
+  shouldComponentUpdate(nextProps) {
+    // Only if the props are different
+    return (
+      nextProps.position !== this.props.position ||
+      nextProps.rotation !== this.props.rotation ||
+      nextProps.midRotation !== this.props.midRotation ||
+      nextProps.perspective !== this.props.perspective ||
+      nextProps.scaleDown !== this.props.scaleDown ||
+      nextProps.scaleFurther !== this.props.scaleFurther ||
+      nextProps.wingSpan !== this.props.wingSpan ||
+      nextProps.spacing !== this.props.spacing ||
+      nextProps.children !== this.props.children
+    );
+  }
+
+  render() {
+    const {
+      scroll,
+      position,
+      rotation,
+      midRotation,
+      perspective,
+      scaleDown,
+      scaleFurther,
+      wingSpan,
+      spacing,
+      onSelect,
+    } = this.props;
+
+    const opacity = scroll.interpolate({
+      inputRange: [
+        position - 3,
+        position - 2,
+        position - 1,
+        position,
+        position + 1,
+        position + 2,
+        position + 3,
+      ],
+      outputRange: [0, 1, 1, 1, 1, 1, 0],
+    });
+
+    const style = {
+      transform: [
+        { perspective },
+        {
+          translateX: scroll.interpolate({
+            inputRange: [
+              position - 2,
+              position - 1,
+              position,
+              position + 1,
+              position + 2,
+            ],
+            outputRange: [
+              spacing + wingSpan,
+              spacing,
+              0,
+              -spacing,
+              -spacing - wingSpan,
+            ],
+          }),
+        },
+
+        {
+          scale: scroll.interpolate({
+            inputRange: [
+              position - 2,
+              position - 1,
+              position,
+              position + 1,
+              position + 2,
+            ],
+            outputRange: [scaleFurther, scaleDown, 1, scaleDown, scaleFurther],
+          }),
+        },
+        {
+          rotateY: scroll.interpolate({
+            inputRange: [
+              position - 2,
+              position - 1,
+              position - 0.5,
+              position,
+              position + 0.5,
+              position + 1,
+              position + 2,
+            ],
+            outputRange: [
+              `-${rotation}deg`,
+              `-${rotation}deg`,
+              `-${midRotation}deg`,
+              '0deg',
+              `${midRotation}deg`,
+              `${rotation}deg`,
+              `${rotation}deg`,
+            ],
+          }),
+        },
+      ],
+    };
+
+    return (
+      <View pointerEvents='box-none' style={styles.container}>
+        <TouchableWithoutFeedback onPress={() => onSelect(position)}>
+          <Animated.View style={{ ...style, opacity }}>
+            {this.props.children}
+          </Animated.View>
+        </TouchableWithoutFeedback>
+      </View>
+    );
+  }
+}
+
+export default Item;
