diff --git a/node_modules/react-native-ux-cam/android/build.gradle b/node_modules/react-native-ux-cam/android/build.gradle
index be49e65..bfa4cf2 100644
--- a/node_modules/react-native-ux-cam/android/build.gradle
+++ b/node_modules/react-native-ux-cam/android/build.gradle
@@ -1,11 +1,15 @@
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
+    	google()
+    	maven {
+           url "https://maven.google.com"
+    	}
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.android.tools.build:gradle:4.1.3'
     }
 }
 
@@ -28,6 +32,10 @@ android {
 
 repositories {
     mavenCentral()
+    google()
+    maven {
+        url "https://maven.google.com"
+    }
 }
 
 dependencies {
diff --git a/node_modules/react-native-ux-cam/android/build.gradle~ b/node_modules/react-native-ux-cam/android/build.gradle~
new file mode 100644
index 0000000..9c5ff2e
--- /dev/null
+++ b/node_modules/react-native-ux-cam/android/build.gradle~
@@ -0,0 +1,36 @@
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.1.3'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+    }
+    lintOptions {
+        abortOnError false
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.facebook.react:react-native:+'
+    implementation files('libs/uxcam.jar')
+}
\ No newline at end of file
diff --git a/node_modules/react-native-ux-cam/ios/RNUxcam.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist b/node_modules/react-native-ux-cam/ios/RNUxcam.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist
new file mode 100644
index 0000000..8a5e054
--- /dev/null
+++ b/node_modules/react-native-ux-cam/ios/RNUxcam.xcodeproj/xcuserdata/mrinalmaheshwari.xcuserdatad/xcschemes/xcschememanagement.plist
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>RNUxcam.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>133</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
