diff --git a/node_modules/singular-react-native/android/build.gradle b/node_modules/singular-react-native/android/build.gradle
index a4d1b64..e1ef6a6 100644
--- a/node_modules/singular-react-native/android/build.gradle
+++ b/node_modules/singular-react-native/android/build.gradle
@@ -35,12 +35,12 @@ repositories {
     }
     mavenCentral()
     jcenter()
-    maven { url 'http://maven.singular.net/' }
+    maven { url 'https://maven.singular.net/' }
     google()
 }
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.singular.sdk:singular_sdk:11.+'
+    implementation 'com.singular.sdk:singular_sdk:12.0.6'
     implementation 'com.android.support:support-annotations:28.0.0'
 }
