diff --git a/node_modules/@skele/components/src/viewport/utils.js b/node_modules/@skele/components/src/viewport/utils.js
index 0286df8..9107fe8 100644
--- a/node_modules/@skele/components/src/viewport/utils.js
+++ b/node_modules/@skele/components/src/viewport/utils.js
@@ -15,9 +15,9 @@ export const isInViewport = (
   const viewportEnd = viewportOffset + viewportSize
   const isViewportOffsetBeforeElement = viewportOffset <= elementOffset
   if (isViewportOffsetBeforeElement) {
-    inViewport = elementOffset - preTriggerAreaSize <= viewportEnd
+    inViewport = elementOffset - preTriggerAreaSize + (elementSize * 0.6) <= viewportEnd
   } else {
-    inViewport = elementEnd + preTriggerAreaSize >= viewportOffset
+    inViewport = elementEnd + preTriggerAreaSize - (elementSize * 0.6) >= viewportOffset
   }
   return inViewport
 }
