diff --git a/node_modules/react-native-shared-element/android/build.gradle b/node_modules/react-native-shared-element/android/build.gradle
index aa62c43..e229106 100644
--- a/node_modules/react-native-shared-element/android/build.gradle
+++ b/node_modules/react-native-shared-element/android/build.gradle
@@ -12,7 +12,7 @@ buildscript {
   }
 
   dependencies {
-    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet('kotlinVersion', '1.4.21')}")
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet('kotlinVersion', '1.6.21')}")
   }
 }
 
