diff --git a/node_modules/react-native-render-html/src/HTMLStyles.js b/node_modules/react-native-render-html/src/HTMLStyles.js
index 9ecc2c3..0665900 100644
--- a/node_modules/react-native-render-html/src/HTMLStyles.js
+++ b/node_modules/react-native-render-html/src/HTMLStyles.js
@@ -199,7 +199,7 @@ function cssToRNStyle(
         key
           .split("-")
           .map((item, index) =>
-            index === 0 ? item : item[0].toUpperCase() + item.substr(1)
+            index === 0 ? item : item[0]?.toUpperCase() + item.substr(1)
           )
           .join(""),
         value,
