diff --git a/node_modules/react-native-radial-gradient/android/build.gradle b/node_modules/react-native-radial-gradient/android/build.gradle
index 6a3fde8..c243c90 100644
--- a/node_modules/react-native-radial-gradient/android/build.gradle
+++ b/node_modules/react-native-radial-gradient/android/build.gradle
@@ -45,5 +45,5 @@ allprojects {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
