diff --git a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java
index c8565c3..3f1802f 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java
+++ b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java
@@ -574,8 +574,7 @@ public class RNGestureHandlerModule extends ReactContextBaseJavaModule {
     UIManagerModule uiManager = getReactApplicationContext().getNativeModule(UIManagerModule.class);
     final int rootViewTag = uiManager.resolveRootTagFromReactTag(ancestorViewTag);
     if (rootViewTag < 1) {
-      throw new JSApplicationIllegalArgumentException("Could find root view for a given ancestor with tag "
-              + ancestorViewTag);
+      return;
     }
     synchronized (mRoots) {
       for (int i = 0; i < mRoots.size(); i++) {
