diff --git a/node_modules/react-native-tus-client/RNTusClient.podspec b/node_modules/react-native-tus-client/RNTusClient.podspec
index b855287..52a2baa 100644
--- a/node_modules/react-native-tus-client/RNTusClient.podspec
+++ b/node_modules/react-native-tus-client/RNTusClient.podspec
@@ -13,7 +13,6 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/*.{h,m}'
 
   s.ios.deployment_target = '8.0'
-
-  s.dependency 'React-Core'
-  s.dependency 'TUSKit'
+  s.dependency 'React'
+  s.dependency 'TUSKit', '~> 1.4.2'
 end
diff --git a/node_modules/react-native-tus-client/android/build.gradle b/node_modules/react-native-tus-client/android/build.gradle
index 509d7c6..b0d5d4b 100644
--- a/node_modules/react-native-tus-client/android/build.gradle
+++ b/node_modules/react-native-tus-client/android/build.gradle
@@ -4,11 +4,15 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
+        google()
+    	maven {
+           url "https://maven.google.com"
+    	}
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:4.2.2'
     }
 }
 
@@ -31,11 +35,15 @@ android {
 
 repositories {
     mavenCentral()
+    google()
+    maven {
+        url "https://maven.google.com"
+    }
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile group: 'org.jetbrains', name: 'annotations', version: '13.0'
-    implementation 'io.tus.java.client:tus-java-client:0.4.2'
-    implementation 'io.tus.android.client:tus-android-client:0.1.9'
+    implementation 'com.facebook.react:react-native:+'
+    implementation group: 'org.jetbrains', name: 'annotations', version: '13.0'
+    implementation 'io.tus.java.client:tus-java-client:0.4.3'
+    implementation 'io.tus.android.client:tus-android-client:0.1.10'
 }
diff --git a/node_modules/react-native-tus-client/android/src/main/java/com/vinzscam/rntusclient/RNTusClientModule.java b/node_modules/react-native-tus-client/android/src/main/java/com/vinzscam/rntusclient/RNTusClientModule.java
index b047dd7..0cd073c 100644
--- a/node_modules/react-native-tus-client/android/src/main/java/com/vinzscam/rntusclient/RNTusClientModule.java
+++ b/node_modules/react-native-tus-client/android/src/main/java/com/vinzscam/rntusclient/RNTusClientModule.java
@@ -125,6 +125,7 @@ public class RNTusClientModule extends ReactContextBaseJavaModule {
       this.uploadId = uploadId;
       client = new TusClient();
       client.setUploadCreationURL(new URL(endpoint));
+      client.enableRemoveFingerprintOnSuccess();
 
       SharedPreferences pref = getReactApplicationContext().getSharedPreferences("tus", 0);
       client.enableResuming(new TusPreferencesURLStore(pref));
