diff --git a/node_modules/react-native-image-resizer/ios/RCTImageResizer/RCTImageResizer.m b/node_modules/react-native-image-resizer/ios/RCTImageResizer/RCTImageResizer.m
index d014831..6225063 100644
--- a/node_modules/react-native-image-resizer/ios/RCTImageResizer/RCTImageResizer.m
+++ b/node_modules/react-native-image-resizer/ios/RCTImageResizer/RCTImageResizer.m
@@ -9,7 +9,7 @@
 
 #if __has_include(<React/RCTBridgeModule.h>)
 #import <React/RCTBridgeModule.h>
-#import <React/RCTImageLoader.h>
+#import <React-RCTImage/React/RCTImageLoader.h>
 #else
 #import "RCTBridgeModule.h"
 #import "RCTImageLoader.h"
