diff --git a/node_modules/singular-react-native/android/build.gradle b/node_modules/singular-react-native/android/build.gradle
index a4d1b64..f008d29 100644
--- a/node_modules/singular-react-native/android/build.gradle
+++ b/node_modules/singular-react-native/android/build.gradle
@@ -35,7 +35,7 @@ repositories {
     }
     mavenCentral()
     jcenter()
-    maven { url 'http://maven.singular.net/' }
+    maven { url 'https://maven.singular.net/' }
     google()
 }
 